cpp/include/cudf/utilities/span.hpp

```diff
@@ -159,9 +159,9 @@ struct host_span : public cudf::detail::span_base<T, Extent, host_span<T, Extent
   // Copy construction to support const conversion
   template <typename OtherT,
             std::size_t OtherExtent,
-            typename std::enable_if<(Extent == OtherExtent || Extent == dynamic_extent) &&
-                                      std::is_convertible_v<OtherT (*)[], T (*)[]>,
-                                    void>::type* = nullptr>
+            typename std::enable_if_t<(Extent == OtherExtent || Extent == dynamic_extent) &&
+                                        std::is_convertible_v<OtherT (*)[], T (*)[]>,
+                                      void>* = nullptr>
```

Comment:
Might as well go all the way. Or, can just use `CUDF_ENABLE_IF` macro. 

```suggestion
            std::enable_if_t<(Extent == OtherExtent || Extent == dynamic_extent) &&
                                        std::is_convertible_v<OtherT (*)[], T (*)[]>>* = nullptr>
```

Reply:
updated.